### PR TITLE
test: add tests to project

### PR DESCRIPTION
**Pull request by DorMosheZarka**
Downloaded and installed the necessary test packages (`@playwright/test`, `mocha`, `chai`) and configured the `playwright.config.ts` file by specifying browser types, timeouts, retries, and the test directory.

### CODE REVIEW

File: test/e2e/example.test.ts
Comment (noammu) [inline test/e2e/example.test.ts]: Does is work?

**Review by noammu — CHANGES_REQUESTED** [review]

File: test-results/.last-run.json
Comment (noammu) [inline test-results/.last-run.json]: Add line

File: test/e2e/example.test.ts
Comment (noammu) [inline test/e2e/example.test.ts]: You checked that it is working,great.
Now lets delete it as it is not relevant :)

**Review by noammu — CHANGES_REQUESTED** [review]

File: package.json
Comment (noammu) [inline package.json]: In Codux we are only testing on Chromium as much as I know so no need to the whole package of playwright with all the data for all browsers.
So lets install only the ```playwright-chromium``` package.

**Review by noammu — CHANGES_REQUESTED** [review]
The title is not right.

File: package.json
Comment (matangeorgi) [inline package.json:25]: Why did you remove prettier from the project? and how is it related to this PR?

File: playwright.config.ts
Comment (matangeorgi) [inline playwright.config.ts]: Please don't set these parameters here: timeout, retries, headless, viewport.

**Review by matangeorgi — COMMENTED** [review]
Please change your title to be according to conventional commits.

File: package.json
Comment (DorMosheZarka) [reply package.json:25]: I didn’t. For some reason, the installation of Playwright moved it a few lines up.

**Review by noammu — APPROVED** [review]
Wait for Matan's approval too

**Comment by matangeorgi** [comment]
lets merge this into the PR that implements the tests, unless you checked out from this branch, and in this case, just close this PR.

**Comment by noammu** [comment]
The reason to merge this PR into the PR that implements the tests is because we don't want to add code that no-one use.
Its called "הכנה למזגן".
Also in Codux: For example, there would probably not be any PR of lets say a driver that no one uses.
So the same here.
So its good to have this PR but not to merge to main (as there are no tests) but to the PR of the tests.

**Pull request closed without merging**
